scripts/histmakers/mw_with_mu_eta_pt.py

```diff
@@ -393,12 +374,6 @@ def build_graph(df, dataset):
 
     df = df.Define("deltaPhiMuonMet", "std::abs(wrem::deltaPhi(goodMuons_phi0,MET_corr_rec_phi))")
 
-    if auxiliary_histograms: 
```

Comment:
The argument against keeping this is that you could just specify different axes and accomplish the same thing, right?

Reply:
In this particular case I think the selection was slightly different (no dphi cut, no mt, so you can apply them at will), so unless you can customise it from command line (which I don't recommend for this purpose) changing axes is not sufficient. Otherwise I would agree with your argument

Reply:
I just deleted the histogramming, I didn't delete the option or the modification of the cuts (at least that was my intention)

Reply:
Ok, but then I'll need to check the code more carefully, I am not sure it can currently make use of the nominal histogram after customising the axes to achieve the same, maybe